http-gateway/service/updateResource_test.go

```diff
@@ -229,3 +232,107 @@ func TestRequestHandlerUpdateResourcesValues(t *testing.T) {
 		})
 	}
 }
+
+func TestRequestHandlerUpdateResourcesValuesWithOnlyContent(t *testing.T) {
+	deviceID := test.MustFindDeviceByName(test.TestDeviceName)
+	switchID := "1"
+	type args struct {
+		accept            string
+		contentType       string
+		deviceID          string
+		resourceHref      string
+		resourceInterface string
+		resourceData      interface{}
+		ttl               time.Duration
```

Comment:
`resourceInterface ` and `ttl` are unused

Reply:
removed